### ISSUE TITLE
'tar: Unrecognized archive format' when attempting to download latest release, dl.gliderlabs.com returns 404 error

### ISSUE THREAD

**Issue by jwigley** [gliderlabs/sigil#23]
Attempting to download the latest sigil based on instructions in the README.

```
$ curl https://dl.gliderlabs.com/sigil/latest/$(uname -sm|tr \  _).tgz \
    | tar -zxC /usr/local/bin
tar: Unrecognized archive format
```

Checking from a browser, https://dl.gliderlabs.com/ returns:

```
Heroku | No such app

There is no app configured at that hostname.
Perhaps the app owner has renamed it, or you mistyped the URL.
```

As a workaround, downloading the latest release from [here](https://github.com/gliderlabs/sigil/releases) works fine.

**Comment by dmytroleonenko** [comment]
The same issue here

**Comment by josegonzalez** [comment]
Closed by #39.
